### ISSUE TITLE
Group 

### ISSUE THREAD

**Issue by BobDL** [HRNet/Higher-HRNet-Human-Pose-Estimation#7]
Hi,
  Thanks for your great job and the open source. I test the pretrained(w32-512)model on the coco set. And for most the results are good. But part of them, the keypoints are grouped  in obviously wrong way.
Is there any opportunity to improve the group?

Thanks
![image](https://user-images.githubusercontent.com/51008026/69773661-60038500-11ce-11ea-9dfe-1ab8b07e3c7b.png)

**Comment by bowenc0221** [comment]
As far as I know, it is hard to improve this kind of error simply using post-processing. Probably you can try different grouping methods other than AE.

**Comment by BobDL** [comment]
> As far as I know, it is hard to improve this kind of error simply using post-processing. Probably you can try different grouping methods other than AE.

Thanks for your reply. Have you tried the paf  grouping method?

**Comment by bowenc0221** [comment]
No, we haven't. We only tried AE in our work since our focus is not on the grouping part. But using better grouping should definitely help.

**Comment by jialee93** [comment]
@BobDL I have tried the PAF-like (proposed by OpenPose) grouping method and pre-trained   HRNet-w42 with Focal L2 loss based on this repo [Simple Pose](https://github.com/jialee93/Improved-Body-Parts) but only get 63 AP on COCO test-dev dataset for now. Multi-scale inference is used during evaluation.

**Comment by massir-rpi** [comment]
Hi @BobDL I am encountering a similar issue. Did you find a good way to improve grouping?
